lib/src/handleErrors.ts

```diff
@@ -3,37 +3,38 @@ import { ExchangeError } from "./error";
 
 /**
  * Display error on LL if LL did not do it. Then throw error back to the live app in case they need to react to it
- * @param walletAPI 
- * @param error 
+ * @param walletAPI
+ * @param error
  */
 export function handleErrors(walletAPI: WalletAPIClient<any>, error: any) {
-  const { message, cause } = error as { message: string; cause: { name: string; message: string; } };
+  const { message, cause } = error as {
+    message: string;
+    cause: { name: string; message: string };
+  };
 
   const ignoredErrorNames = new Set([
-      "WrongDeviceForAccount",
-      "WrongDeviceForAccountPayout",
-      "WrongDeviceForAccountRefund",
-      "CancelStepError",
-      "ConfirmStepError",
-      "SwapCompleteExchangeError",
-      "DisconnectedDevice",
-      "NotEnoughBalance",
-      "NotEnoughGas",
-      "CompleteExchangeError",
-      "TransportStatusError",
+    "SwapCompleteExchangeError",
+    "DisconnectedDevice",
+    "NotEnoughBalance",
+    "NotEnoughGas",
+    "CompleteExchangeError",
+    "TransportStatusError",
   ]);
 
   const ignoredMessages = new Set([
-      "User refused"
+    "User refused",
+    "WrongDeviceForAccount",
+    "WrongDeviceForAccountPayout",
+    "WrongDeviceForAccountRefund",
```

Comment:
This will not work. 
WrongDeviceForAccount is in the `names` list because you find it in 
error.cause.name and not error.message. 

Those list aren't related to business logic but pure object access properties